tictactoe/src/App.js

```diff
@@ -66,27 +65,40 @@ export default function Game() {
     setCurrentMove(nextMove);
   }
 
-  const moves = history.map((squares, move) => {
-    let description;
-    if (move > 0) {
-      description = 'Go to move #' + move;
-    } else {
-      description = 'Go to game start';
-    }
-    return (
-      <li key={move}>
-        <button onClick={() => jumpTo(move)}>{description}</button>
-      </li>
-    );
-  });
+  const moves = history.map((squares, move) => (
+    <option key={move} value={move}>
+      {move === 0 ? 'Go to game start' : `Go to move #${move}`}
+    </option>
+  ));
 
   return (
     <div className="game">
       <div className="game-board">
         <Board xIsNext={xIsNext} squares={currentSquares} onPlay={handlePlay} />
       </div>
+
       <div className="game-info">
-        <ol>{moves}</ol>
+        {/* Navigation buttons */}
+        <div>
+          {/* backward button */}
+          <button className="bwd_button" onClick={() => jumpTo(currentMove - 1)} disabled={currentMove === 0}>
+            Backward
+          </button>
+          <span style = {{ margin: '0 5px'}}></span>
```

Comment:
It's better to put the css exclusively in the index.css file.

Reply:
Alright, thanks for the suggestion.